From: SJ Park <sj@kernel.org>
Date: Wed, 28 Oct 2026 07:45:51 -0700
Subject: [PATCH] ==== various fixes and cleanups ====

mm/damon: various fixes and cleanup of code, test, and docs

Fix and cleanup various parts of DAMON including its code, test, and
documentats.

Remove unnecessary initializations in damon_new_ctx() with patch 1 from
Adrian Huang (Lenovo).  Align MMU notifier ranges when clearing PTE/PMD
young bits with patches 2 and 3 from Shubhang Kaushik (Ampere) and
Aveline Noir.  Avoid DAMON_STAT idle time percentile overflow on 32b-bit
architectures with patch 4 from Aveline Noir.  Let sysfs.py test does
not hang when it is executed under interactive terminal with patch 5
from Jaeyeong Lee.  Fix typos, grammartical errors and styles in
DAMON_RECLAIM comments and DAMON_LRU_SORT user documents with patches 6
and 7 from Manuel.  Finally, document how to disable watermarks of
DAMON_RECLAIM with patch 8 from Jinjie Ruan.

Notes that are not need to be git history.  This is a collection of
DAMON patches that individually posted by contributors and resposted to
be merged into mm.git.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_r55a6ffp | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_r55a6ffp

diff --git a/hkml_cv_bogus/hkml_cv_bogus_r55a6ffp b/hkml_cv_bogus/hkml_cv_bogus_r55a6ffp
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

